### DISCUSSION TITLE
Sense: maximization

### DISCUSSION THREAD

**Discussion by iaae** [snl-dakota/dakota#133] [Q&A]
I have tested a few methods, and the maximize objective function is neglected and some of them proceed with sense: minimization.
Are there any guidelines for such maximization control for all different methods?

If we know a method can only perform minimization, then it is possible to reverse the objective function value for the maximization process.

thanks!

**Comment by jadamstephens** [comment]
All the maximization control does is multiply the objective function by -1. You could do that in your driver. If you happen to remember any of the optimizers that aren't treating the maximize sense correctly, it would be helpful.
